cmd/gapit/flags.go

```diff
@@ -334,4 +334,10 @@ type (
 		Gapir GapirFlags
 		Out   string `help:"output file to save the profiling result"`
 	}
+
+	GraphVisualizationFlags struct {
+		Gapis         GapisFlags
+		Gapir         GapirFlags
```

Comment:
Do we really need Gapir flags here? (I don't expect that we have to kick off a replay)

Reply:
No, We do not really need. I am expecting also do not to kick off a replay. I removed the Gapir flag.
:+1: